magic_the_gathering/game_state.py

```diff
@@ -213,3 +209,18 @@ def zone_position_to_one_hot_vector(self, zone: ZonePosition = None) -> np.ndarr
         if zone is not None:
             one_hot_vector[zone.value] = 1
         return one_hot_vector
+
+    def hide_information_to_current_player(self):
+        clone_game_state = deepcopy(self)
+        for player_index in range(self.n_players):
+            if player_index == clone_game_state.current_player_index:
+                clone_game_state.zones[ZonePosition.LIBRARY][player_index] = clone_game_state.zones[
+                    ZonePosition.LIBRARY
+                ][player_index]
+                clone_game_state.zones[ZonePosition.HAND][player_index] = clone_game_state.zones[ZonePosition.HAND][
+                    player_index
+                ]
```

Comment:
Does this do anything?

Reply:
Well spotted! I removed these lines, indeed only the condition "player_index != clone_game_state.current_player_index" is relevant here :slightly_smiling_face: